pkg/cosmos/params/utils.go

```diff
@@ -0,0 +1,35 @@
+package params
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+
+	"github.com/cometbft/cometbft/crypto"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+	"golang.org/x/crypto/ripemd160" //nolint: staticcheck
+)
+
+// Creates a bech32 address from a hex-encoded secp256k1 public key.
+func CreateBech32Address(pubKey, accountPrefix string) (string, error) {
```

Comment:
not a big deal but iirc, this function/logic is duplicated in the keystore in core right? is it worth consolidating into one function across both repos?